metals/src/main/scala/scala/meta/internal/metals/codelenses/RunTestCodeLens.scala

```diff
@@ -74,8 +73,15 @@ final class RunTestCodeLens(
     } yield {
       val classes = buildTargetClasses.classesOf(buildTargetId)
       if (connection.isScalaCLI && path.isAmmoniteScript) {
-        scalaCliCodeLenses(textDocument, buildTargetId, classes, distance)
+        scalaCliCodeLenses(
+          textDocument,
+          buildTargetId,
+          classes,
+          distance,
+          isJVM,
+        )
```

Comment:
so this only works for scala cli right now because it can run native and js, right?

Reply:
It works also for Bloop, but for sbt BSP it seems to only detect the JVM classes.

Reply:
Mill gets a bunch of exceptions, but I don't want to dig into that yet.

Reply:
Newest Mill doesn't get the exceptions, though it hangs for JS, not sure why. But that's not really related to this PR. We can turn it off for mill, but then we might forget about it.